### PR TITLE
Implemented timer thing

### PR DESCRIPTION
**Pull request by marcsello**

### CODE REVIEW

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java [hunk 1]
```diff
@@ -23,6 +30,23 @@ public void followSpeed() {
...
+
+	@Override
+	public void run() {
+		try {
+			while (true) {
+				Thread.sleep(1000);
+				followSpeed();
+			}
+		} catch (InterruptedException e) {
+			// It's fine
```
Comment (danielpetho) [inline train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java]: untreated exception!

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java [hunk 2]
```diff
@@ -23,6 +30,23 @@ public void followSpeed() {
...
 	}
+
+
+
+
+	@Override
+	public void run() {
+		try {
+			while (true) {
+				Thread.sleep(1000);
```
Comment (danielpetho) [inline train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java]: túl kevés az 1 sec

**Review by danielpetho — CHANGES_REQUESTED** [review]
update!

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java (hunk 2 above)
Comment (marcsello) [reply train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java]: is 1.001 sec okay?

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java (hunk 1 above)
Comment (marcsello) [reply train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java]: k. Added logging instead

**Review by danielpetho — APPROVED** [review]

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java (hunk 2 above)
Comment (danielpetho) [reply train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java]: ja

**Pull request merged**